src/components/Image/docs/examples.ts

```diff
@@ -3,21 +3,40 @@ export const importExample = `import { Image } from 'wix-ui-tpa/Image';`;
 export const absoluteUrlExample = `
 <Image
   src="https://m.media-amazon.com/images/M/MV5BZGMwOGIwZjUtOWQ1OS00YWRjLWJmZGMtN2Y1OWQ3ZDYwYTM3XkEyXkFqcGdeQXVyNzU1NzE3NTg@._V1_.jpg"
-  width="300"
-  height="250"
+  width="480"
+  height="360"
   alt="Garfield smiles and puts his hand over chest"
 />
 `;
 
 export const relativeUriExample = `
 <Image
   src="c5f754_dd75514d14fa4057b4f4a6cc8ce7add3~mv2.jpg"
-  width="300"
-  height="250"
+  width="480"
+  height="360"
```

Comment:
Just alining the image to original dimensions to emphasize the diff in the resizing example